config/variables/stacks/appeals-service/prod.hcl

```diff
@@ -1,5 +1,13 @@
 locals {
   appeals_feature_flags = [
+    {
+      name    = "appeal-form-v2"
+      enabled = true

```

Comment:
can we turn off completely in prod for now

Reply:
Done